docs/about/release_notes.rst

```diff
@@ -31,6 +27,119 @@ Release Notes
 Highlighted changes
 ===================
 
+Version 9.0
+-----------
+
+New workflow: Evaluate ensemble
+###############################
+
+In this version of ERT we have introduced a new simulation mode called: Evaluate ensemble.
+The main reason for this is that in the past, the Ensemble experiment simulation mode has had
```

Comment:
```suggestion
The main reason for this is that in the past, the Ensemble experiment simulation mode has had
```

The primary reason is that the `Ensemble experiment` simulation mode has historically served two distinct purposes: conducting sensitivity analyses and doing manual updates.